src/App/Pages/Accounts/LoginPage.xaml

```diff
@@ -74,7 +74,8 @@
                 </Grid>
             </StackLayout>
             <StackLayout Padding="10, 0">
```

Comment:
i'd move the visibility binding to this stack layout

Reply:
Moved.